sdcm/cluster.py

```diff
@@ -322,15 +322,42 @@ def __init__(self, name, ssh_login_info=None, base_logdir=None, node_prefix=None
         self.is_enterprise = None
         self.replacement_node_ip = None  # if node is a replacement for a dead node, store dead node private ip here
         self._set_prometheus_paths()
+        self.distro = None
+
+    def get_distro(self):
+        if self.distro:
+            return self.distro
+
+        result = self.remoter.run('grep CentOS /etc/redhat-release', ignore_status=True)
+        if result.exit_status == 0:
+            self.distro = 'centos'
+
+        result = self.remoter.run('cat /etc/issue', ignore_status=True)
+        if 'Ubuntu 14.04' in result.stdout:
+            self.distro = 'ubuntu14'
```

Comment:
Generally you should avoid using strings since it is easy to make mistake and code completion will not work
I would use Enum instead Like OS.ubuntu14, OS.ubuntu16, etc.

Reply:
I prefer to use enum, but I found `enum` isn't installed by default in my machine.
So I didn't use that.